v1/api.go

```diff
@@ -147,9 +150,30 @@ func (c *Client) ExecRequest(_ context.Context, request interface{}) (interface{
 	return result, nil
 }
 
+// ExecNoResponseRequest executes a client operation that does not return a model, query or write metadata.
+//func (c *Client) ExecNoResponseRequest(_ context.Context, request interface{}) (string, error) {
+func (c *Client) ExecNoResponseRequest(_ context.Context, request interface{}) (interface{}, error) {
+	typeOf := reflect.TypeOf(request)
+	valueOf := reflect.ValueOf(request)
+
+	_, ok := typeOf.MethodByName("Execute")
+	if !ok {
+		return nil, errors.New("ExecNoResponseRequest failed: no Execute method on interface")
+	}
+
+	values := valueOf.MethodByName("Execute").Call([]reflect.Value{})
+	if !values[1].IsNil() {
+		return nil, values[1].Interface().(error)
+	}
+
+	result := values[0].Interface()
```

Comment:
I'm confused by this.  If this returns no response, why does this method return a result?  Is this just the HTTP response?  If so, I think we don't need to return it.

Reply:
I had tried to get it to just return err, but I couldn't get the test to run with an error.  So I ended up attaching the HTTP response as a result and the test just checks to see that it's not nil.

You're absolutely right, the result has no bearing on the  test, I just couldn't get the test to compile/run.

Reply:
Added!

If I update the test to:
```
		err := testClient.System().GarbageCollect(writeOpts.Ctx())
		require.NoError(t, err)
```

and I get: `./system_test.go:16:7: no new variables on left side of :=`

...or am I supposed to just run the test without assigning the output to a variable?  i.e. `testClient.System().GarbageCollect(writeOpts.Ctx())`